src/legacy/server/saved_objects/saved_objects_mixin.js

```diff
@@ -50,7 +50,7 @@ export function savedObjectsMixin(kbnServer, server) {
     getSavedObjectsClient: {
       assign: 'savedObjectsClient',
       method(req) {
-        return req.getSavedObjectsClient();
+        return req.getSavedObjectsClient({ includeHiddenTypes: ['secretType'] });
```

Comment:
We definitely don't want to merge with this, or else there are no secrets.

Reply:
omg, good call. This would have been bad!